packages/skeleton-svelte/src/components/pagination/anatomy/first-trigger.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts" module>
+	import type { HTMLAttributes } from '@/internal/html-attributes';
+	import type { PropsWithElement } from '@/internal/props-with-element';
+
+	export interface PaginationFirstTriggerProps extends PropsWithElement<'button'>, HTMLAttributes<'button'> {}
+</script>
+
+<script lang="ts">
+	import { RootContext } from '../modules/root-context';
+	import { classesPagination } from '@skeletonlabs/skeleton-common';
+	import { mergeProps } from '@zag-js/svelte';
+
+	const props: PaginationFirstTriggerProps = $props();
+
+	const pagination = RootContext.consume();
+
+	const { element, children, ...rest } = $derived(props);
+
+	const attributes = $derived(
+		mergeProps(
+         {
+            onclick:  pagination().goToFirstPage()
+         },
+			{
+				class: classesPagination.nextTrigger,
+			},
+			rest,
+		),
+	);
```

Comment:
You can actually put this next to class:

```suggestion
	const attributes = $derived(
		mergeProps(
			{
				class: classesPagination.nextTrigger,
				onclick:  pagination().goToFirstPage()
			},
			rest,
		),
	);
```